nixos/modules/services/audio/gonic.nix

```diff
@@ -41,7 +41,11 @@ in
       description = "Gonic Media Server";
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
-      serviceConfig = {
+      serviceConfig = let
```

Comment:
Honestly, I think it's fair to not make this conditional. It wouldn't hurt too much to have it "less-hardened" always.